backend/tests/caseNote.test.js

```diff
@@ -187,6 +188,38 @@ describe('case note management', () => {
         await expect(handleCreateCaseNoteForm(req, res)).rejects.toThrowError(error)
       })
 
+      it('should return the specific error in case of a 400 response', async () => {
+        const req = {
+          ...mockCreateReq,
+          params: { offenderNo },
+          body: {
+            type: 'P',
+            subType: 'PI',
+            date: '20/07/2020',
+            hours: '10',
+            minutes: '10',
+            text: 'test',
+          },
+        }
+        const error400 = {
```

Comment:
Not a massive issue but you've tested this slightly differently above.

Reply:
Fixed it. I did that originally and moved so I could change quicker as things weren't working